src/app/business/block/buckets.html

```diff
@@ -27,16 +27,6 @@
         </p-column>
         <p-column field="LocationConstraint" header="Default Backend" ></p-column>
         <p-column field="CreateTime" header="Created At"></p-column>
-        <p-column field="versionEnabled" header="Versioning Enabled">
-            <ng-template pTemplate="body" let-bucket="rowData">
-                <p *ngIf="bucket.versionEnabled">
-                    <span class="version-text">Yes</span>
-                </p>
-                <p *ngIf="!bucket.versionEnabled">
-                    <span class="version-text">No</span>
```

Comment:
should we just keep this for non-versioned buckets ?

Reply:
As per discussions we will remove this from the UI currently.